src/fd-icon.scss

```diff
@@ -0,0 +1,22 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-icon;
+
+[class*="#{$block}"] {
```

Comment:
We're trying to move away from `[class*="#{$block}"] `, can this just reference the block like on line 16?

Reply:
Changed

Reply:
Reverted. 
We can't remove this part, because most of the icons doesn't have common unique class,
all of them looks like `.sap-icon--washing-machine` or `.sap-icon--wrench`.

It could be a good idea to add this class in future, for example as in the font-awesome:
```
<i class="fas fa-arrow-right"></i>
<i class="fas fa-arrow-left"></i>
```
I think this update shouldn't be a part of this PR.

Reply:
That sounds reasonable! Can we make a followup issue for that? All places using an icon should probably be something like: `<i class='fd-icon fd-icon--iconName" />`

Reply:
@jbadan could you review please this issue :)
https://github.com/SAP/fundamental-styles/issues/1794